### DISCUSSION TITLE
Spare parts (PCB + level shifter + 6 wires cables) in US

### DISCUSSION THREAD

**Discussion by choumarin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#370] [General]
I have 4 sets of spare hardware, that includes the PCB, the level shifter and the 6 wires cables (male + female).
Just add the nodemcu.
![image](https://user-images.githubusercontent.com/7562982/188342952-d517d440-7c57-47be-82a1-faf6cecca2eb.png)
Free if you pay for shipping or pickup in the bay area.

**Comment by rosetown-kelly** [comment]
Hi @choumarin. Do you still have these parts? Let me know the best way to coordinate if you are still looking to get rid of them. Looking forward to taking on this project!

**Reply by choumarin** [reply]
I still do. reach out to me at ----

**Reply by GHLurker** [reply]
Hello @choumarin . Are any of these still available?

**Reply by choumarin** [reply]
Yes, I still have some. ping me at ---

**Reply by pelican1997** [reply]
Hi, @choumarin. Do you still have any of these available?

**Reply by choumarin** [reply]
Hi, sorry, I don't have anything left
